utility/index.jsx

```diff
@@ -58,13 +59,13 @@ export const transformations = {
       />
       <div style={{ display: 'inline-block', verticalAlign: 'top' }}>
         {constants.heroes[field] ? constants.heroes[field].localized_name : ''}
-        {row.match_id ?
+        {!row.rank && (row.match_id ?
```

Comment:
I think I'm going to factor this whole thing into a component. Please look it over once I push updates.